app/src/js/components/ReconciliationReports/reconciliation-report.js

```diff
@@ -131,9 +110,15 @@ class ReconciliationReport extends React.Component {
     return { granuleFilesOnlyInCumulus, granuleFilesOnlyInCmr };
   }
 
+  handleCardClick (e, index) {
+    e.preventDefault();
+    this.setState({ activeIdx: index });
+  }
+
   render () {
     const { reconciliationReports } = this.props;
     const { reconciliationReportName } = this.props.match.params;
+    const { activeIdx } = this.state;
 
     const record = reconciliationReports.map[reconciliationReportName];
```

Comment:
Nothing to be done here, but I'm noting what an unfortunate choice of name "map"  we did it all over the place.

Reply:
Agreed